frontend/src/employee-frontend/api/applications.ts

```diff
@@ -49,8 +53,8 @@ export async function getApplication(
       })),
       attachments: data.attachments.map((attachment) => ({
         ...attachment,
-        updated: new Date(attachment.updated),
-        receivedAt: new Date(attachment.receivedAt)
+        updated: LocalDate.fromSystemTzDate(new Date(attachment.updated)),
+        receivedAt: LocalDate.fromSystemTzDate(new Date(attachment.receivedAt))
```

Comment:
`LocalDate.parseIso` seems to be the used for ISO string -> LocalDate conversion. Should we use that too?

Reply:
That was my first guess but resulted to a hard-to-catch bug - the reason being that the function can't parse timestamps, only iso dates

Reply:
Shouldn't we return dates from the backend then? It seems confusing to me that we're explicitly asking for extra information (the time components in the timestamp) and then immediately throwing it away in the API call deserialization code.

Or, if we can't change the backend payload and just don't want to show the time components to users, it sounds to me like a formatting concern that belongs to the components showing the data.

Reply:
Imho all dates should be serialized as iso strings in response payloads. The time component might be relevant in another use case so I would not get rid of it.

> just don't want to show the time components to users, it sounds to me like a formatting concern that belongs to the components showing the data.

I think this is the right answer; pass the value around as a Date and render it without the time component. Using LocalDate _everywhere_ in the frontend might not be a viable long-term solution anyway because there will most likely be use cases where we need the time too.

Reply:
Yes I think you guys are absolutely correct here 👍 I was just blindly stuck in the idea of _always_ using `LocalDate` over `Date`.